### ISSUE TITLE
Warning for tether module when building with Webpack

### ISSUE THREAD

**Issue by yuters** [furqanZafar/react-selectize#33]
```
WARNING in ./~/react-selectize/src/Tether.js
There is another module with an equal name when case is ignored.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Rename module if multiple modules are expected or use equal casing if one module is expected.

WARNING in ./~/react-selectize/src/tether.js
There is another module with an equal name when case is ignored.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Rename module if multiple modules are expected or use equal casing if one module is expected.
```

Changing the class name `Tether` for something else seems to solve the issue. But I'm still not sure of the best way to solve this.

**Comment by furqanZafar** [comment]
hmm, the warning makes sense i am creating a react-class named `Tether` which uses an external plugin named `tether`, so i guess i will rename my component to `ReactTether` or `TetherWrapper`

**Comment by furqanZafar** [comment]
should be fixed in 0.7.0, there are some breaking changes related to theming & stylesheets though.
https://github.com/furqanZafar/react-selectize/blob/develop/CHANGELOG.md

**Comment by yuters** [comment]
Thanks! :)
